### PR TITLE
Refactor(catalog-dashboard): Use a UI dashboard lib component

### PR DESCRIPTION
**Pull request by xmano**
Refactoring Catalog app dashboard #3292

### CODE REVIEW

File: apps/catalog/catalog/src/app/dashboard/dashboard.component.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿<layout-dashboard hideSearchbar>
+  <aside fxLayout="column" fxLayoutAlign="space-between">
+    <mat-nav-list fxFlex>
+      <a mat-list-item routerLink="home" routerLinkActive="active">
+        <mat-icon matListIcon svgIcon="dashboard"></mat-icon>
+        <span matLine><b>Dashboard</b></span>
+      </a>
+      <a mat-list-item routerLink="titles" routerLinkActive="active">
```
Comment (Hugo-Hontans) [inline apps/catalog/catalog/src/app/dashboard/dashboard.component.html:8]: Normally on current develop the routerLink should be `title` and not `titles`.
Can you verify this route plz ?

Comment (xmano) [reply apps/catalog/catalog/src/app/dashboard/dashboard.component.html]: Corrected.

File: apps/catalog/catalog/src/app/dashboard/dashboard.component.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿<layout-dashboard hideSearchbar>
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/dashboard/dashboard.component.html:1]: I looked into the code and it seems that there is no input hideSearchbar in the layout-dashboard. You can remove it here an din the festival app too.

Comment (xmano) [reply apps/catalog/catalog/src/app/dashboard/dashboard.component.html]: I removed `hideSearchbar` from both catalog & festival.

**Comment by xmano** [comment]
Closes #3292

**Pull request merged**